### ISSUE TITLE
Check numerical issues

### ISSUE THREAD

**Issue by mravanelli** [speechbrain/speechbrain#133]
It could be great to add a quick check (Nan or Inf) to the training loss function at the end of each batch. We should stop training if some numerical issues are met.  It could also be great to add some advice like: "to debug it, use pytorch checkgrad..."
